nomi-ceu/assets/questbook/lang/zh_cn.lang

```diff
@@ -2472,1057 +2472,1057 @@ nomifactory.quest.normal.db.780.desc=一旦你获得了你的第一个§6下界
 nomifactory.quest.normal.db.781.title=通货能量
 nomifactory.quest.normal.db.781.desc=§6通货能源炉§r是整合包游戏中期的主要能量来源。用一个§a宝石印刻校准§r和一个自给自足的§b深度怪物学习§r搭配安装，这个能源炉可以烧掉§6钻石§r来产生大量的§eRF能量§r。%n%n就像其他§b热力膨胀§r能源炉一样，§6通货能源炉§r可以被赋予升级套件，以产生更多的能量，并获得更多的§a增强槽位§r。
 
-# db 782 of mode normal
```

Comment:
What happened to the translations for quest id 782?

Reply:
Large Turbines -> High Amperage Energy Hatches
There was no translation for High Amperage Energy Hatches before. I'll get this one up after I finish looking at the other quests.